packages/graphback-core/src/runtime/GraphbackContext.ts

```diff
@@ -11,5 +11,6 @@ export interface GraphbackServiceConfigMap {
  * GraphQL context interface according to Graphback runtime layer format
  */
 export interface GraphbackContext {
-  graphback: GraphbackServiceConfigMap
+  graphback: GraphbackServiceConfigMap,
+  selectedFields: string[]
```

Comment:
Is this okay?

Reply:
I think @wtrocki mentioned putting into an options object on the context

```ts
export interface GraphbackContext {
  graphback: GraphbackServiceConfigMap
  graphbackOptions: {
    selectedFields: string[]
  }
}
```

I can see why this is a good idea - to keep the top level context small.

Reply:
I would be looking to have typescript definitions of our context so it will be strongly typed and easy to work with. We have generics already just need to start with some base:

context.graphback - services
context.graphbackOptions - request level arguments
context.graphbackConfig - configuration (we are passing this around in services but we can have it mounted in context as well)

Reply:
Isn't confusing to have `graphback` and `graphbackOptions` on top level object?

Reply:
We are free to propose different namespaces. 
```
graphback.services.Tastk
graphback.config
graphback.optons
```

Reply:
+1 on having
 ```
context.graphback.options.selectedFields
context.graphback.services.Entity
context.graphback.config.crud.find
```

Reply:
@machi1990 would you make this small changes for us?

Reply:
> +1 on having
> 
> ```
> context.graphback.options.selectedFields
> context.graphback.services.Entity
> context.graphback.config.crud.find
> ```

This should be the case now for `services` and `options` from my recent push. I did not want to bring in the `config` change yet.If we can open an issue for that, it can be looked post this PR.

Reply:
Config is redundant - it was used as example. 
All good with options and services. 
@craicoverflow WDYT?

Reply:
Agreed that there is no need for both options and config.

Reply:
I think there is a need to have The `options` key as a strongly typed interface though, so it can be passed down to the data providers instead of the full context.

At present the entire context is getting passed which I  think is overkill.

See https://github.com/aerogear/graphback/pull/1509/files#r444043195